src/main/java/org/broadinstitute/hellbender/tools/walkers/concordance/Concordance.java

```diff
@@ -0,0 +1,244 @@
+package org.broadinstitute.hellbender.tools.walkers.concordance;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextComparator;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.VariantProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.Optional;
+
+@CommandLineProgramProperties(
+        summary = "Evaluate a vcf against a vcf of validated (true) variants",
+        oneLineSummary = "Evaluate a vcf against a vcf of validated (true) variants",
+        programGroup = VariantProgramGroup.class
+)
+
+/**
+ *
+ * This tool evaluates a vcf against a validated truth vcf. We assume that the truth vcf only contains PASS variants.
+ * The summary statistics (# true positives, # false positives, # false negatives, sensitivity, precision)
+ * are reported as a summary tsv (--summary). Also reported as a tsv (--table) are evaluated variants of interest
+ * (i.e. true positives, false positives, false negatives) and their annotations (ref allele, alt allele, minor allele fraction, etc.)
+ * for further statistical analysis e.g. with an R script
+ *
+ * java -jar gatk.jar Concordance -V na12878-eval.vcf --truth na12878-truth.vcf --table table.tsv --summary summary.tsv --evalSampleName NA12878
+ *
+ * Created by tsato on 1/30/17.
+ */
+public class Concordance extends VariantWalker {
+    @Argument(doc = "truth vcf (tool assumes all sites in truth are PASS)", fullName= "truth", shortName = "T")
+    protected File truth;
+
+    @Argument(doc = "TO BE IMPLEMENTED", fullName= "confidence", shortName = "C", optional = true)
+    protected File highConfidenceRegion;
+
+    @Argument(doc = "A table of evaluation variants of interest and their basic annotations",
+            fullName="table", shortName = "O")
+    protected File table;
+
+    @Argument(doc = "A table of summary statistics (true positives, sensitivity, etc.)",
+            fullName="summary", shortName = "S")
+    protected File summary;
+
+    @Argument(doc = "sample name of the eval variants", fullName ="evalSampleName", shortName = "sample")
+    protected String evalSampleName;
+
+    // TODO: output a vcf of false positives (and negative) sites?
+    // TODO: take a low confidence region where a false positive there is not counted (masked in dream?)
+    // TODO: what about other variant types besides SNP and INDEL?
+
+    // we count evaluation status in these variables
+    private long snpTruePositives = 0;
```

Comment:
Yea I had a change of heart ... `++` looks much better than `.increment()` haha. But yea for consistency's sake I will use `MutableLongs`.